www/install/sql/centreon/Update-DB-20.04.0-beta.1.sql

```diff
@@ -10,3 +10,10 @@ DELETE FROM `cb_type` WHERE `type_shortname` = 'correlation';
 UPDATE topology SET topology_url_opt = '&o=svcOV_pb' WHERE topology_page = 20204;
 UPDATE topology SET topology_url_opt = '&o=svcOVHG_pb' WHERE topology_page = 20209;
 UPDATE topology SET topology_url_opt = '&o=svcOVSG_pb' WHERE topology_page = 20212;
+
+-- Update nagios_server to add gorgone connection
+ALTER TABLE `nagios_server` ADD `gorgone_communication_type` enum('1','2') NOT NULL DEFAULT '1' AFTER `centreonconnector_path`;
+ALTER TABLE `nagios_server` CHANGE `ssh_port` `gorgone_port` INT(11) NULL;
```

Comment:
just a comment -> why not 5556 instead of null, as it's the default port for zmq

Reply:
https://github.com/centreon/centreon/pull/8184/files#diff-e867c4fbf07f9192685567130cae39d5R2

Reply:
https://github.com/centreon/centreon/blob/master/www/install/createTables.sql#L1629